subprojects/parseq-tracevis/build.gradle

```diff
@@ -11,10 +12,10 @@ apply plugin: 'com.github.node-gradle.node'
 
 node {
   // Version of node to use.
-  version = '6.11.2'
+  version = '20.8.1'
```

Comment:
Wow 14 major versions!  Very nice!

Reply:
I was surprised that nothing totally broke after this many majors :)